_test/configuration.test.php

```diff
@@ -0,0 +1,59 @@
+<?php
+/**
+ * General tests for the findologicxmlexport plugin
+ *
+ * @group plugin_findologicxmlexport
+ * @group plugins
+ */
+
+//======================================================================
+// PLUGIN CONFIGURATION TESTS
+//======================================================================
+
+require_once(__DIR__ . '/../DokuwikiXMLExport.php');
+require_once(__DIR__ . '/../admin.php');
+require_once(__DIR__ . '/helper.php');
+
+class configuration_plugin_findologicxmlexport_test extends DokuWikiTest
+{
+    public function setUp()
+    {
+        $helper = new helper;
```

Comment:
How does this even work without the parentheses? :fearful: Anyway, as discussed elsewhere you could just make the methods static.